quasar_migrations/src/m20231013_065827_create_contract_table.rs

```diff
@@ -0,0 +1,56 @@
+use sea_orm_migration::prelude::*;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        manager
+            .create_table(
+                Table::create()
+                    .table(Contract::Table)
+                    .if_not_exists()
+                    .col(
+                        ColumnDef::new(Contract::Address)
+                            .string()
+                            .not_null()
+                            .primary_key(),
+                    )
+                    .col(
+                        ColumnDef::new(Contract::Hash)
+                            .string()
+                            .not_null()
+                            .unique_key(),
+                    )
+                    .col(ColumnDef::new(Contract::Key).string().not_null())
+                    .col(ColumnDef::new(Contract::Type).string().not_null())
+                    .col(
+                        ColumnDef::new(Contract::LastModified)
+                            .integer()
+                            .not_null(),
+                    )
+                    .to_owned(),
+            )
+            .await
+    }
+
+    async fn down(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        manager
+            .drop_table(Table::drop().table(Contract::Table).to_owned())
+            .await
+    }
+}
+
+#[derive(DeriveIden)]
+enum Contract {
+    #[sea_orm(iden = "contracts")]
+    Table,
+    Address,
+    Hash,
+    Key,
+    Type,
+    LastModified,
+    // TODO: @geofmureithi Soroban contracts can be introspected.
```

Comment:
I'd be against keeping TODOs in commited code. We can keep things to do as Github issues and stick to using TODOs only in local code.

What do you think?

Reply:
Sure, I will create an issue, this is related to the Discord question, I was waiting to know the next step.

Reply:
Documented here: https://github.com/eigerco/quasar/issues/42

Reply:
Can you remove this line then?

Reply:
```suggestion
```